spec/scanner_spec.rb

```diff
@@ -31,18 +31,19 @@
     subject { scanner.scan.to_a }
 
     it "should match unpatched gems to their advisories" do
+      ids = subject.map { |result| result.advisory.id }
+      expect(ids).to include('OSVDB-89025')
```

Comment:
So this test was broken in two ways:
- `OSVDB-89026` was the advisory ID that was being used, and that is on `actionpack` and does not apply to the `unpatched_gems` Gemfiles.  It looks some past refactoring trimmed that Gemfile down significantly.
- That problem was not causing a test failure though because as @jychen7 pointed out above, the assertions in the test were not checking for the expected advisory ID, which in the later test is ignored.

Fixed this by:
- Changing the Gemfile so it uses an earlier `activerecord 3.2.10`.
- looking for OSVDB-89025 which does affect `activerecord 3.2.10`.
- also changing the ignore test to NOT expect OSVDB-89025.

Altogether this seemed like the minimal change here.